Cargo.toml

```diff
@@ -66,6 +66,8 @@ http-body = { workspace = true }
 bytes = { workspace = true }
 pin-project-lite = { workspace = true }
 cfg-if = {workspace = true }
+blake3 = { workspace = true }
+tempfile = "3.18.0"
```

Comment:
Should this be in workspace dependencies?

Reply:
It should, I'll fix.